src/textual/widgets/_select.py

```diff
@@ -305,6 +305,46 @@ def __init__(
         self._value = value
         self._setup_variables_for_options(options)
 
+    @classmethod
+    def from_values(
+        cls,
+        options: Iterable[SelectType],
+        *,
+        prompt: str = "Select",
+        allow_blank: bool = True,
+        value: SelectType | NoSelection = BLANK,
+        name: str | None = None,
+        id: str | None = None,
+        classes: str | None = None,
+        disabled: bool = False,
+    ):
+        """Initialize the Select control with options specified by an iterable of
+        of any type (string, integer, etc.).  The string representation in the
+        control is generated by this method using the built-in str().
+
+        Args:
+         options: Options to select from.
+         prompt: Text to show in the control when no option is select.
+         allow_blank: Allow the selection of a blank option.
+         value: Initial value (should be one of the values in `options`).
+         name: The name of the select control.
+         id: The ID of the control the DOM.
+         classes: The CSS classes of the control.
+         disabled: Whether the control is disabled or not.
+        """
+        options_iterator = [(str(o), o) for o in options]
```

Comment:
You only replaced part of abbreviations here :)

```suggestion
        options_iterator = [(str(option), option) for option in options]
```